app/models/extends/file.js

```diff
@@ -16,7 +16,7 @@ const fileProvider = nconf.get('filedb');
 
 const FileSchema = new mongoose.Schema({
   // buffer limit 16MB when attached to document!
-  name: {type: String},
+  name: {type: String, default: 'no-name'},
```

Comment:
why default name ?

Reply:
I needed something in it so I can link log events to specific files.

But now that you mentioned it, what is the name fields function at this time anyways? Filename is derived from the sha1 checksum so it is not that.

Reply:
name is original name of that file. filename for storage place is sha1 for now - but could be anything in future so it's not something which actually describe filename for clients.

Reply:
I think name could be even required field - then there doesn't need to be any default value. Can be changed in another PR.